src/main/java/picard/illumina/ClusterDataToSamConverter.java

```diff
@@ -74,6 +74,7 @@
     // TODO: - https://github.com/samtools/hts-specs/pull/119
     private String molecularIndexTag          = "RX";
     private String molecularIndexQualityTag   = "QX";
+    private String molecularIndexDelimiter    = "-";
```

Comment:
`molecularIndexDelimiter` can be yes, but the others can be overwritten with appropriate setter methods.